### PR TITLE
Fix to  - eos_lag_interfaces: Port-channel getting removed

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY
Fix to #64615

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
lag_interfaces.py

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

File: test/integration/targets/eos_lag_interfaces/tests/cli/deleted.yaml [hunk 1]
```diff
@@ -24,6 +25,11 @@
...
 
+- set_fact:
+    expected_config:
+      - name: "Port-Channel10"
+      - name: "Port-Channel5"
+
 - assert:
     that:
-      - "ansible_facts.network_resources.lag_interfaces == []"
+      - "ansible_facts.network_resources.lag_interfaces|symmetric_difference(expected_config)|length == 0"
```
Comment (Qalthos) [inline test/integration/targets/eos_lag_interfaces/tests/cli/deleted.yaml:35]:
```suggestion
      - "ansible_facts.network_resources.lag_interfaces|symmetric_difference(expected_config) == []"
```

File: test/integration/targets/eos_lag_interfaces/tests/cli/overridden.yaml [hunk 2]
```diff
@@ -33,4 +37,4 @@
 
 - assert:
     that:
-      - "ansible_facts.network_resources.lag_interfaces|symmetric_difference(config)|length == 0"
+      - "ansible_facts.network_resources.lag_interfaces|symmetric_difference(expected_config)|length == 0"
```
Comment (Qalthos) [inline test/integration/targets/eos_lag_interfaces/tests/cli/overridden.yaml:40]:
```suggestion
      - "ansible_facts.network_resources.lag_interfaces|symmetric_difference(expected_config) == []"
```

File: test/integration/targets/eos_lag_interfaces/tests/cli/deleted.yaml [hunk 3]
```diff
@@ -1,9 +1,10 @@
 ---
-- include_tasks: reset_config.yml
+- include_tasks: _reset_config.yml
```
Comment (Qalthos) [inline test/integration/targets/eos_lag_interfaces/tests/cli/deleted.yaml:2]: I'm confused by what's going on here... you've added a file named `reset_config.yaml`, but including `_reset_config.yml`?

File: test/integration/targets/eos_lag_interfaces/tests/cli/reset_config.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    expected_config:
+      - name: "Port-Channel10"
+      - name: "Port-Channel5"
+        members:
+          - member: Ethernet1
+            mode: "on"
+
+- assert:
+    that:
+      - "ansible_facts.network_resources.lag_interfaces|symmetric_difference(expected_config)|length == 0"
```
Comment (Qalthos) [inline test/integration/targets/eos_lag_interfaces/tests/cli/reset_config.yaml:26]:
```suggestion
      - "ansible_facts.network_resources.lag_interfaces|symmetric_difference(expected_config) == []"
```

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Comment by GomathiselviS** [comment]
This plugin is no longer maintained in this repository and has been migrated to https://github.com/ansible-collections/arista.eos


IRC: #ansible-network on irc.freenode.net
mailing list: https://groups.google.com/forum/#!forum/ansible-project

**Pull request closed without merging**